lib/topological_inventory/ansible_tower/operations/source.rb

```diff
@@ -26,6 +28,8 @@ def initialize(params = {}, request_context = nil)
         def availability_check
           return if params_missing?
 
+          return if checked_recently?
```

Comment:
Does this implicitly return true? I assume it does.

Reply:
@lindgrenj6 The `return` without a parameter will pass a `nil` back to the caller.  Is that what you are referring to?

Reply:
the `checked_recently?` never returns nil, this line returns nil and caller doesn't care about return value of operation